api/src/main/resources/workbench-api.yaml

```diff
@@ -7280,3 +7329,95 @@ definitions:
         format: int64
       conceptName:
         type: string
+  AuditLogEntriesResponse:
+    type: object
+    description: >
+      Represents the results of a query into the audit dataset in BigQery without
+      filtering or transformation. This response type is a stopgap until we develop
```

Comment:
This comment might feel better placed in a design document or Jira ticket rather than API documentation.

I'm not sure I fully understand why this is considered a stopgap – can you give any examples of higher-level queries or semantic mapping that you would want to build? From my viewpoint, it covers just about everything I would have imagined we'd want for the admin pages.